cfme/automate/explorer/instance.py

```diff
@@ -336,6 +249,89 @@ def delete_if_exists(self):
             self.delete()
 
 
+@attr.s
+class InstanceCollection(BaseCollection):
+
+    ENTITY = Instance
+
+    @property
+    def tree_path(self):
+        return self.parent.tree_path
+
+    def create(self, name=None, display_name=None, description=None, fields=None, cancel=False):
+        add_page = navigate_to(self, 'Add')
+        fill_dict = {
+            k: v
+            for k, v in {
+                'name': name,
+                'display_name': display_name,
+                'description': description,
+                'fields': fields,
+            }.items()
+            if v is not None}
+        add_page.fill(fill_dict)
+        if cancel:
+            add_page.cancel_button.click()
+            add_page.flash.assert_no_error()
+            add_page.flash.assert_message('Add of new Automate Instance was cancelled by the user')
```

Comment:
**Optional**
Consider `flash.assert_success_message()`, which calls `assert_no_error` and `assert_message`